### DISCUSSION TITLE
SOS integration with Elastic security

### DISCUSSION THREAD

**Discussion by mfbasha** [Security-Onion-Solutions/securityonion#12375] [Unsupported Versions]
What if i need to teploy Security onion as one of the log sources to Elastic SIEM\fleet  
every type of alerts , flows, events in addition to the raw log to be forwarded and indexed on elastic node , each in diferent index so we can create more advanced dashboards and use cases on Kibana

**Comment by reyesj2** [comment]
You just want to setup Security Onion to send its logs elsewhere? You could setup a logstash pipeline to forward events https://docs.securityonion.net/en/2.4/logstash.html#original-event-forwarding
